docs/source/prereqs.md

```diff
@@ -0,0 +1,231 @@
+# Prerequisites
+
+Let's start with Prerequisites:
+
+
+#### Mac
+
+> **Homebrew**
+> For macOS, we recommend using [Homebrew](https://brew.sh) to manage the prereqs.
+> ```shell
+> $ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+> $ brew --version
+> Homebrew 2.5.2
+> ```
+
+> The Xcode command line tools will be installed as part of the Homebrew installation.  Once Homebrew is ready, installing the necessary prerequisites is very easy:
+
+> **Git**
+> Install the latest version of [git](https://git-scm.com/downloads) if it is not already installed.
+
+> ```shell
+> $ brew install git
+> $ git --version
+> git version 2.23.0
+> ```
+
+> **cURL**
+> Install the latest version of [cURL](https://curl.haxx.se/download.html) if it is not already installed.
+> ```shell
+> $ brew install curl
+> $ curl --version
+> curl 7.64.1 (x86_64-apple-darwin19.0) libcurl/7.64.1 (SecureTransport) LibreSSL/2.8.3 zlib/1.2.11 nghttp2/1.39.2
+> Release-Date: 2019-03-27
+> ```
+
+> **wget**
+> Install the latest version of [wget](https://www.gnu.org/software/wget/) if it is not already installed.
+> ```shell
+> $ brew install wget
+> $ wget -V
+> GNU Wget 1.20.3 built on darwin19.0.0.
+> ```
+
+> **Docker**
+> Install the latest version of [Docker](https://docs.docker.com/get-docker/) if it is not already installed. 
+> ```shell
+> $ brew cask install --appdir="/Applications" docker
+> ```
+> Docker Desktop must be launched to complete the installation so be sure to open the application after installing it:
+> ```shell
+> $ open /Applications/Docker.app
+> ```
+
+> Once installed, confirm the latest versions of both `docker` and `docker-compose` executables were installed.
+> ```shell
+> $ docker --version
+> Docker version 19.03.12, build 48a66213fe
+> $ docker-compose --version
+> docker-compose version 1.27.2, build 18f557f9
+> ```
+
+
+> **GoLang** 
+> Install the latest version of [go](https://golang.org/doc/install) if it is not already installed.
+> ```shell
+> $ brew install go
+> $ go version
+> go1.14.6 darwin/amd64
+>```
+
+> **JQ**
```

Comment:
Only needed if doing config transactions, not needed for the two basic tutorials that you recommend starting with.

Reply:
@denyeart true - if the install section is just for the two tutorials then we could add the JQ install instructions in the Tutorials that reference config transactions, here for instance: https://hyperledger-fabric.readthedocs.io/en/latest/channel_update_tutorial.html#setup-the-environment

But, I thought the install section was to enable the users to be able to use any tutorial without more installs.